src/framework-ondata.js

```diff
@@ -52,17 +52,19 @@ module.exports = function(payload, socketIndex) {
          if ( ! _.isFunction( this.nodeInstance.wrapper.fRunUpdater ) ) { 
 
              // Don't have a wrapper fRunUpdater 
+             profiler.stopEvent(eventStart);
```

Comment:
I did not do this originally because the intent was to have the event set up time + updater time equal the total time executing the framework-ondata.  However, in thinking this through with your other comment below, it does simplify things if we make the event time the total time spent in this function (stopping it in a finally block) and then moving the updater time to being around the fRunUpdater call.  We lose no information that way since the event set up time is the event time - updater time.   As long as David does not object, I will make this change.